src/main.rs

```diff
@@ -93,6 +131,27 @@ async fn main() -> quilkin::Result<()> {
             quilkin::run(config, vec![]).await
         }
 
+        Commands::Manage {
+            port,
+            admin_port,
+            provider,
+        } => {
+            let provider = match provider {
+                ProviderCommands::Agones {
+                    gameservers_namespace,
+                    config_namespace,
+                } => std::sync::Arc::from(
+                    quilkin::xds::provider::AgonesProvider::new(
+                        gameservers_namespace,
```

Comment:
Not for this PR (and I'm sure you've thought the same), but I expect long term, we'll want to support multiple namespaces, particularly for Agones GameServers (in dev, a namespace per developer is not uncommon).